src/MakeLocalizeFunction.js

```diff
@@ -1,7 +1,10 @@
 /**
+ * Convert the localization object into a function in case we need to support nested keys.
  *
- * @param {object}  localization
- * @returns {Function}
+ * @param {object} localization the language object,
```

Comment:
`{type}` => `{Type}`

Reply:
I thought about it, but saw lowercase everywhere so left it. I can change this.

Reply:
Yep would be appreciated while we are at it 😛

Reply:
done